cypress/plugins/index.js

```diff
@@ -19,4 +20,7 @@
 module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+  // eslint-disable-next-line no-param-reassign
+  config.env.FRONTEND_URL = process.env.FRONTEND_URL;
+  return config;
```

Comment:
Could we splat `process.env` into the export in a new `config.cypress.js` file? https://docs.cypress.io/guides/guides/environment-variables#Option-1-configuration-file That would let us share `.env` every where.

Reply:
Went ahead and changed that!